spec/duration.html

```diff
@@ -1387,10 +1388,10 @@ <h1>AdjustRoundedDurationDays ( _years_, _months_, _weeks_, _days_, _hours_, _mi
             [[Nanoseconds]]: _nanoseconds_
             }.
         1. Let _timeRemainderNs_ be ? TotalDurationNanoseconds(0, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_, 0).
-        1. Let _direction_ be ! Sign(_timeRemainderNs_).
+        1. Let _direction_ be ! ℝ(Sign(𝔽(_timeRemainderNs_))).
```

Comment:
Maybe we should consider removing Sign after all. As far as I can tell, it never actually operates on Number values in the Temporal spec text, so the whole justification for adding it and using it in `Math.sign()` when merging into 262 doesn't exist. Maybe this could be something like "Let _direction_ be the mathematical value whose sign is the sign of _timeRemainderNs_ and whose magnitude is 1." (while accounting for 0 with an assertion)?

Reply:
I'll deal with Sign later, I think.

Reply:
→ #1593